src/pal/src/CMakeLists.txt

```diff
@@ -276,8 +276,29 @@ endif(CMAKE_SYSTEM_NAME STREQUAL Darwin)
 
 if(CMAKE_SYSTEM_NAME STREQUAL FreeBSD)
   if(CLR_CMAKE_USE_SYSTEM_LIBUNWIND)
-    find_library(UNWIND unwind)
-  endif()
+    if(PAL_CMAKE_PLATFORM_ARCH_AMD64)
```

Comment:
Could you please rather extract the code that you've copied here from the if(CMAKE_SYSTEM_NAME STREQUAL Linux) into a new separate if(CMAKE_SYSTEM_NAME STREQUAL Linux OR CMAKE_SYSTEM_NAME STREQUAL FreeBSD) so that we don't have two copies?